C4SGSvc/src/main/resources/application.properties

```diff
@@ -5,7 +5,7 @@
 # Set here configurations for the database connection
 
 # Connection url for the database "netgloo_blog"
-spring.datasource.url = jdbc:mysql://localhost:3306/C4SG
+spring.datasource.url = jdbc:mysql://localhost:3306/c4sg
```

Comment:
This change was made by PraniDahal @PranilDahal  and I just cherry pick this commit, and will change back later, thanks for pointing out.